python/ray/WebUI.ipynb

```diff
@@ -32,7 +70,9 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": true,
+    "hideCode": false,
+    "hidePrompt": false
```

Comment:
Let's undo all of these `hideCode` and `hidePrompt` changes. I don't think they help.